awx_collection/tools/vars/resolution.yml

```diff
@@ -0,0 +1,6 @@
+---
+name_to_id_endpoint_resolution:
+  webhook_credential: credentials
+  project: projects
+  inventory: inventories
+  organization: organizations
```

Comment:
I don't think we should continue to rely on name_to_id, and encoding the logic in the generator is going in the wrong directions. I would suggest that `param_to_endpoint` can just be called, and there is no need for an entire pipeline for producing this call pattern.